src/glang/codegen/builtin_types.py

```diff
@@ -472,8 +486,15 @@ def ir_type(self) -> str:
 
     @property
     def ir_mangle(self) -> str:
-        member_mangle = "".join(member.ir_mangle for _, member in self.members)
-        return f"__S{member_mangle}"
+        member_types = (m_type for _, m_type in self.members)
+        # <builtin-type> ::= u <source-name> [<template-args>]  # vendor extended type
+        # FIXME binutils rejects the above :(.
+        # `u <source-name>` works and so does `<source-name> <template-args>`,
+        # so fall back to the latter until we can fix it (seems like a quick
+        # fix in gcc/libiberty/cp-demangle.c).
+        return (
+            f"{mangle_source_name('AnonymousType')}{mangle_template_args(member_types)}"
+        )
```

Comment:
Damn, good documenting. How about the llvm tooling?

Reply:
```
$ llvm-cxxfilt _Z3foou1TIiiE
_Z3foou1TIiiE
```

Nope :(

Surprisingly, as this was an amendment to the spec proposed by a clang engineer!